packages/amplify-category-hosting/__tests__/lib/S3AndCloudFront/configuration-manager.test.js

```diff
@@ -0,0 +1,108 @@
+jest.mock('inquirer');
+const inquirer = require('inquirer');
+
+jest.mock('../../../lib/S3AndCloudFront/helpers/configure-Website');
+const configureWebsite = require('../../../lib/S3AndCloudFront/helpers/configure-Website'); 
+jest.mock('../../../lib/S3AndCloudFront/helpers/configure-CloudFront');
+const configureCloudFront = require('../../../lib/S3AndCloudFront/helpers/configure-CloudFront'); 
+jest.mock('../../../lib/S3AndCloudFront/helpers/configure-Publish');
+const configurePublish = require('../../../lib/S3AndCloudFront/helpers/configure-Publish'); 
+
+const configurationManager = require('../../../lib/S3AndCloudFront/configuration-manager'); 
+
+describe('configuration-manager', () => {
+    const mockAmplifyMeta = {
```

Comment:
Does this need to be of this shape or can it be a simple object

Reply:
I'm using a real meta data.